pkg/api/object.go

```diff
@@ -221,11 +220,47 @@ func (h *Handler) Listbuckets(w http.ResponseWriter, r *http.Request) {
 	FormatJsonResponse(w, sortedBuckets)
 }
 
-func handleConnectionLookupFailure(err error, action string, w http.ResponseWriter) {
-	logger.Errorf("failed to get %s: %+v", action, err)
-	if util.IsEtcdKeyNotFound(err) {
-		w.WriteHeader(http.StatusNotFound)
-	} else {
+// GetBucket gets the bucket from the object store in this cluster.
+// GET
+// /objectstore/buckets/{USER_ID}
```

Comment:
should the `bucketname` query param be added to this comment? is `{USER_ID}` needed or is it a typo?